### ISSUE TITLE
Pothole detection system

### ISSUE THREAD

**Issue by arpy8** [abhisheks008/DL-Simplified#276]
### Deep Learning Simplified Repository (Proposing new issue)

:red_circle: **Project Title** : Pothole detection system
:red_circle: **Aim** : To create a machine learning model that would detect potholes on road
:red_circle: **Dataset** : https://www.kaggle.com/datasets/atulyakumar98/pothole-detection-dataset
:red_circle: **Approach** : The pothole detection model is built on top of the YOLOv8 architecture, which is a state-of-the-art object detection algorithm. YOLOv8 utilizes a single neural network to simultaneously predict bounding boxes and classify objects within those boxes.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Arpit Sengar
- GitHub Profile Link : https://github.com/arpy8
- Email ID : arpitsengar99@gmail.com
- Approach for this Project : The pothole detection model is built on top of the YOLOv8 architecture, which is a state-of-the-art object detection algorithm. YOLOv8 utilizes a single neural network to simultaneously predict bounding boxes and classify objects within those boxes.
- What is your participant role? (Mention the Open Source program) SSOC'23

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
You have to use atleast 3 deep learning models/methods for this project. Compare them according to their accuracy scores and provide the best fitted model among the implemented ones.

Issue assigned to you @arpy8

**Comment by LokeshYarramallu** [comment]
i have started working on this can oyu assaign this to me

**Comment by MattiaFerrarini** [comment]
Full name : Mattia Ferrarini
GitHub Profile Link : https://github.com/MattiaFerrarini
Email ID : mattiaferrarini22@gmail.com
Approach for this Project : Perform all the necessary exploratory data analysis, dataset cleaning and feature engineering. Employ various CNN models which can be found in keras, such as InceptionV3 and ResNet. Compare performances based on accuracy and (if appreciated) AUC.
What is your participant role? (Mention the Open Source program) SWOC 2024

**Comment by abhisheks008** [comment]
Issue assigned to you @MattiaFerrarini
